server/src/StorageServiceS3.ts

```diff
@@ -83,4 +84,52 @@ export default class StorageService implements IStorageService {
 
     return returnDataList;
   }
+
+  public async putBucketLifecycleConfiguration(days: string) {
+    const params: AWS.S3.PutBucketLifecycleConfigurationRequest = {
+      Bucket: envModule.AWS_BUCKETNAME,
+      LifecycleConfiguration: {
+        Rules: [ /* required */
+          {
+            Prefix: "",
+            Status: "Enabled",
+            Expiration: {
+              Days: 3560,
+            },
+            ID: "S3toDEEP_ARCHIVE",
+            Transitions: [
+              {
+                Days: +days,
+                StorageClass: "DEEP_ARCHIVE",
+              },
+            ],
+          },
+        ],
+      },
+    };
+
+    const putObjectPromise = await s3.putBucketLifecycleConfiguration(params).promise();
+    console.log(putObjectPromise);
+    return putObjectPromise;
+  }
+
+  public async getBucketLifecycleConfiguration(): Promise<LifecycleRule | undefined> {
+
+    const params: AWS.S3.GetBucketLifecycleConfigurationRequest = {
+      Bucket: envModule.AWS_BUCKETNAME,
+    };
+
+    const getBucketLifecycleConfiguration = await s3.getBucketLifecycleConfiguration(params).promise();
```

Comment:
variable name should be noun, not verb.

Reply:
replaced getBucketLifecycleConfiguration with bucketLifecycleConfiguration